### ISSUE TITLE
Remove non API functions

### ISSUE THREAD

**Issue by yinleon** [SMAPPNYU/youtube-data-api#23]
I had originally added some convenience functions to deal with in-the-wild URLs and to collect comments, but these functions aren't actually part of the YouTube API.

I'd like to set a deprecation date on those function. The utils might be added to gists for convenience or ported to a separate library of helper functions.

Thoughts? @mabrownnyu

**Comment by mabrownnyu** [comment]
Yeah I think I'd agree with that. It probably makes more sense to separate them off into a package than keep them on as random utilities since they don't specifically relate to the functions of the YouTube API itself.
